frontend/src/components/Checkbox/Checkbox.types.ts

```diff
@@ -0,0 +1,10 @@
+interface CheckboxProps {
+  label?: string;
+
+  checked: boolean;
```

Comment:
JFI, there is good practice to sort mandatory and optional props: so at the top you can see all the props you have to pass and then the rest of possible props